airflow/models/dagrun.py

```diff
@@ -1314,7 +1314,7 @@ class DagRunNote(Base):
     __tablename__ = "dag_run_note"
 
     user_id = Column(Integer, nullable=True)
-    dag_run_id = Column(Integer, primary_key=True, nullable=False)
+    dag_run_id = Column(Integer, primary_key=True, name="dag_run_note_pkey", nullable=False)
```

Comment:
Does this work?

Reply:
Ah good catch, it does not. We need a PrimaryKeyConstraint in `__table_args__` instead.

Reply:
It works. Unless you have a composite key then it does not work

Reply:
You can test with sqlite:
`breeze --backend sqlite`
`airflow db init`
`airflow db shell`
`.schema dag_run_note`

Reply:
Yeah. You are right @uranusjr . The naming convention was what was making the name appear as if this is working. I will change it